lib/runtime-core/build.rs

```diff
@@ -1,13 +1,13 @@
-use blake2b_simd::blake2bp;
+use blake3;
```

Comment:
Unless I'm mistaken about how it works in build.rs, `use item;` never does anything meaningful anymore. Due to uniform paths, if something is reachable in a certain way via use, it is also reachable that way in other paths throughout the module. So since `use blake3;` works, `blake3::Hasher` must therefore already work in the module without having to specifically import it via use. rustc should honestly emit a warning there, but it doesn't yet.